content/articles/record-editor.markdown

```diff
@@ -44,14 +44,24 @@ From the <label>Add a Record</label> menu select a DNS record type, and click on
 
 ![Select Record Type](/files/rec-editor-new-record.png)
 
-The creation page varies depending on the record type you selected. In most cases, you'll be asked to provide a _Name_, a _TTL_, and the data (or content) for the record.
+## Simple record form
```

Comment:
Can we call it "Simple editor" as in the UI?

Reply:
Missed that. +1 on it.